File: src/main/java/com/natia/secretmod/core/ItemPickupEvent.java
package com.natia.secretmod.core;

import com.natia.secretmod.utils.ItemDiff;
import net.minecraft.item.ItemStack;
import net.minecraftforge.fml.common.eventhandler.Event;

public class ItemPickupEvent extends Event {
    private ItemDiff stack;
    public ItemPickupEvent(ItemDiff stack) {
        this.stack = stack;
    }

    public ItemDiff getItemDiff() {
        return stack;
    }
}

File: src/main/kotlin/natia/skytweaks/core/ItemPickupEvent.kt
package natia.skytweaks.core

import natia.skytweaks.utils.ItemDiff
import net.minecraftforge.fml.common.eventhandler.Event

class ItemPickupEvent(val itemDiff: ItemDiff) : Event() {
    @JvmName("getItemDiff1")
    fun getItemDiff(): ItemDiff {
        return itemDiff
    }
}
